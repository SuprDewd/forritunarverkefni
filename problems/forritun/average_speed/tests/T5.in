00:00:48 5
00:01:33
00:07:42 13
00:11:20 148
00:13:22
00:19:57
00:21:03 127
00:23:20 33
00:28:13 149
00:35:23 20
00:41:06 183
00:47:56 92
00:48:34
00:52:32
00:53:10 131
01:00:39
01:03:30
01:10:54 178
01:17:16 113
01:20:53 180
01:22:24
01:22:58 99
01:27:32 41
01:31:10 134
01:32:18
01:36:31
01:41:44 149
01:44:57 186
01:52:25 117
01:52:58
01:55:02
01:56:05
01:59:15 115
02:05:24
02:10:02 180
02:16:39 169
02:19:22
02:20:50 5
02:26:34 101
02:31:43 177
02:32:40
02:36:59 158
02:44:29
02:52:33
02:56:53
02:57:53 20
03:04:39
03:06:26 89
03:14:39 94
03:21:50
03:30:02
03:36:27
03:40:19
03:40:48
03:43:19
03:43:53
03:50:52 70
03:53:38 63
04:00:57 24
04:08:48 198
04:12:24
04:18:00 26
04:23:25
04:24:54 171
04:28:17 23
04:33:34
04:40:24 36
04:47:44
04:55:32 10
05:03:26 137
05:09:34
05:11:36
05:17:41 31
05:21:13 164
05:27:53 149
05:30:09 188
05:30:14
05:37:50 69
05:44:21
05:47:58 111
05:53:33 179
05:54:24 13
05:58:52
06:01:31 150
06:08:36 115
06:11:46 20
06:11:47 118
06:16:58
06:25:01
06:32:16 167
06:33:26 55
06:35:57 122
06:43:05 8
06:51:19
06:54:42
07:01:15
07:04:09 43
07:09:56 173
07:11:16
07:19:01
07:25:18 101
07:32:27 56
07:32:59 3
07:40:08
07:47:18 17
07:52:09
08:00:16
08:04:36 153
08:10:59
08:14:20
08:17:16 177
08:24:53
08:30:31 44
08:36:28
08:42:21 62
08:42:28
08:45:33
08:52:14
08:56:43 70
09:00:46 124
09:02:25 105
09:07:48 24
09:11:06 150
09:17:05 59
09:24:36 129
09:26:02
09:31:31
09:34:26 100
09:39:52
09:41:50 50
09:45:56
09:54:05
09:58:08
10:02:14
10:03:36
10:03:41
10:05:16 194
10:06:27
10:07:19
10:14:32 71
10:17:19 138
10:19:14 99
10:25:18
10:33:20
10:33:52 95
10:41:11
10:48:43
10:54:33 180
11:02:14 27
11:06:22
11:12:34 69
11:15:45 39
11:18:03
11:25:25
11:33:19 59
11:37:42
11:39:38 111
11:45:45 69
11:48:03 9
11:54:34 176
11:55:03
11:57:26 174
12:03:24 187
12:03:36
12:08:39
12:10:02
12:18:20 50
12:22:32
12:28:35 192
12:34:20 187
12:39:07 174
12:44:37 15
12:51:26
12:59:27
13:02:59
13:04:31 172
13:06:17 161
13:10:41
13:18:52 52
13:25:05 73
13:29:10
13:29:37
13:33:17 20
13:39:17
13:43:22
13:49:29
13:57:41 24
14:04:16
14:06:50 167
14:12:47 31
14:15:22
14:17:26 15
14:19:18
14:24:32
14:28:14
14:28:40
14:34:29 23
14:39:46 77
14:44:59 66
14:46:39
14:47:28 170
14:52:28
14:58:04 26
15:02:02 150
15:06:25
15:08:01 43
15:12:54 158
15:13:35
15:14:23
15:19:15
15:21:36
15:22:47 179
15:30:36 159
15:30:38
15:36:55
15:38:09
15:45:45
15:51:21
15:54:40
15:57:11 17
15:57:26 120
16:03:29
16:06:17 177
16:08:14
16:08:38
16:11:33
16:13:59 50
16:15:27 49
16:19:37 73
16:26:56
16:28:57 188
16:30:31 1
16:37:02
16:40:43 0
16:47:39
16:47:52 46
16:55:30 141
16:57:31 139
17:02:15
17:05:11
17:12:17 124
17:14:05
17:16:59
17:22:58
17:28:22 17
17:30:55
17:38:25 176
17:41:41 175
17:44:06
17:46:00
17:47:06
17:48:29
17:55:17 7
18:02:51
18:06:22
18:07:15
18:13:38
18:16:43 173
18:22:33 49
18:30:30 176
18:35:43
18:37:25
18:38:57
18:42:08
18:42:58
18:48:23
18:50:11 90
18:54:53 1
19:02:49 15
19:04:24
19:09:47
19:12:14 34
19:16:20 133
19:18:04
19:22:17 53
19:23:50
19:29:38
19:30:14 22
19:37:35 53
19:44:15
19:44:28
19:44:29
19:51:31
19:57:40
20:04:35 77
20:10:18
20:13:52
20:19:22 13
20:24:12
20:24:42
20:27:38 9
20:28:55
20:29:57 192
20:34:22 51
20:37:41 72
20:42:47
20:49:46
20:56:42 126
20:57:14
21:03:39
21:05:56 26
21:08:39
21:16:54 85
21:21:46 184
21:29:17
21:29:21
21:33:55
21:35:36
21:43:56 37
21:46:47
21:50:26
21:51:14 9
21:52:51
21:57:52 187
22:03:57
22:04:49
22:11:22
22:13:56 108
22:18:41 166
22:25:38 30
22:29:12 146
22:33:44 126
22:35:19 190
22:42:43 70
22:44:38 87
22:48:10 153
22:50:14 67
22:58:24 129
23:06:02 182
23:07:53
23:09:55
23:16:39 110
23:20:18
23:25:17
23:29:30
23:36:18 81
23:44:15 18
23:44:41 76
23:50:02 101
23:55:56
24:02:42
24:10:54 124
24:17:04 49
24:19:38 12
24:20:33 93
24:27:16
24:29:29
24:34:00
24:36:46
24:39:32
24:40:16
24:48:33
24:50:10
24:54:39
24:56:35
24:59:12
25:01:56 42
25:06:45 103
25:11:37 55
25:15:15 67
25:19:34
25:20:02
25:27:22 182
25:31:44
25:32:21 78
25:33:02
25:37:46 125
25:45:07 60
25:49:11
25:53:44
25:54:32 9
25:55:17
26:00:19
26:08:12 23
26:09:49
26:12:58 145
26:16:13
26:22:31 4
26:27:09 97
26:31:13
26:37:22
26:38:12
26:39:04 188
26:41:35
26:42:27
26:42:59
26:46:24 176
26:53:14 193
26:56:42
26:58:25 47
27:04:39 110
27:09:51 80
27:12:53
27:16:54 96
27:25:08
27:32:23 89
27:38:14 200
27:43:03 92
27:47:14
27:50:36
27:52:11
27:53:28 119
27:56:18 48
28:02:23
28:04:50 97
28:10:56 107
28:18:49
28:19:21
28:21:37 148
28:28:19 132
28:28:21
28:33:58 11
28:42:05
28:47:16 199
28:54:44
28:56:42
28:57:20
29:01:28 178
29:01:33 88
