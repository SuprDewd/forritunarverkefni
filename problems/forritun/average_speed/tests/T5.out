00:01:33 0.06 km
00:13:22 6.38 km
00:19:57 22.62 km
00:48:34 74.36 km
00:52:32 80.45 km
01:00:39 97.76 km
01:03:30 103.98 km
01:22:24 150.38 km
01:32:18 164.63 km
01:36:31 174.05 km
01:52:58 217.91 km
01:55:02 221.94 km
01:56:05 223.99 km
02:05:24 241.95 km
02:19:22 278.33 km
02:32:40 294.41 km
02:44:29 326.90 km
02:52:33 348.14 km
02:56:53 359.55 km
03:04:39 364.44 km
03:21:50 388.47 km
03:30:02 401.32 km
03:36:27 411.37 km
03:40:19 417.43 km
03:40:48 418.19 km
03:43:19 422.13 km
03:43:53 423.02 km
04:12:24 459.89 km
04:23:25 480.72 km
04:33:34 493.03 km
04:47:44 500.05 km
05:09:34 520.05 km
05:11:36 524.69 km
05:30:14 564.52 km
05:44:21 595.83 km
05:58:52 613.82 km
06:16:58 648.37 km
06:25:01 664.20 km
06:51:19 699.62 km
06:54:42 700.07 km
07:01:15 700.94 km
07:11:16 709.32 km
07:19:01 731.66 km
07:40:08 762.67 km
07:52:09 764.40 km
08:00:16 766.70 km
08:10:59 784.21 km
08:14:20 792.75 km
08:24:53 822.70 km
08:36:28 843.68 km
08:42:28 848.12 km
08:45:33 851.30 km
08:52:14 858.21 km
09:26:02 907.15 km
09:31:31 918.94 km
09:39:52 934.26 km
09:45:56 940.96 km
09:54:05 947.75 km
09:58:08 951.13 km
10:02:14 954.54 km
10:03:36 955.68 km
10:03:41 955.75 km
10:06:27 960.90 km
10:07:19 963.70 km
10:25:18 1004.74 km
10:33:20 1018.00 km
10:41:11 1030.46 km
10:48:43 1042.39 km
11:06:22 1076.54 km
11:18:03 1084.48 km
11:25:25 1089.27 km
11:37:42 1098.72 km
11:55:03 1116.97 km
12:03:36 1141.89 km
12:08:39 1157.63 km
12:10:02 1161.94 km
12:22:32 1191.31 km
12:51:26 1247.31 km
12:59:27 1249.32 km
13:02:59 1250.20 km
13:10:41 1267.46 km
13:29:10 1299.77 km
13:29:37 1300.32 km
13:39:17 1306.78 km
13:43:22 1308.14 km
13:49:29 1310.18 km
14:04:16 1315.55 km
14:15:22 1334.47 km
14:19:18 1336.00 km
14:24:32 1337.31 km
14:28:14 1338.24 km
14:28:40 1338.34 km
14:46:39 1350.35 km
14:52:28 1365.42 km
15:06:25 1393.96 km
15:13:35 1403.26 km
15:14:23 1405.37 km
15:19:15 1418.18 km
15:21:36 1424.37 km
15:30:38 1450.90 km
15:36:55 1467.55 km
15:38:09 1470.81 km
15:45:45 1490.95 km
15:51:21 1505.79 km
15:54:40 1514.58 km
16:03:29 1533.42 km
16:08:14 1544.78 km
16:08:38 1545.96 km
16:11:33 1554.56 km
16:26:56 1575.27 km
16:37:02 1582.74 km
16:47:39 1582.80 km
17:02:15 1604.35 km
17:05:11 1611.15 km
17:14:05 1631.32 km
17:16:59 1637.31 km
17:22:58 1649.68 km
17:30:55 1661.56 km
17:44:06 1680.32 km
17:46:00 1685.86 km
17:47:06 1689.07 km
17:48:29 1693.10 km
18:02:51 1713.82 km
18:06:22 1714.23 km
18:07:15 1714.33 km
18:13:38 1715.07 km
18:35:43 1754.05 km
18:37:25 1759.04 km
18:38:57 1763.53 km
18:42:08 1772.87 km
18:42:58 1775.32 km
18:48:23 1791.20 km
19:04:24 1804.06 km
19:09:47 1805.41 km
19:18:04 1812.19 km
19:23:50 1822.90 km
19:29:38 1828.03 km
19:44:15 1837.14 km
19:44:28 1837.33 km
19:44:29 1837.35 km
19:51:31 1843.56 km
19:57:40 1848.99 km
20:10:18 1862.44 km
20:13:52 1867.01 km
20:24:12 1875.12 km
20:24:42 1875.23 km
20:28:55 1876.06 km
20:42:47 1899.28 km
20:49:46 1907.66 km
20:57:14 1917.10 km
21:03:39 1930.58 km
21:08:39 1936.55 km
21:29:17 1970.07 km
21:29:21 1970.28 km
21:33:55 1984.28 km
21:35:36 1989.44 km
21:46:47 2016.76 km
21:50:26 2019.01 km
21:52:51 2019.74 km
22:03:57 2039.45 km
22:04:49 2042.16 km
22:11:22 2062.57 km
23:07:53 2181.69 km
23:09:55 2187.86 km
23:20:18 2214.98 km
23:25:17 2224.11 km
23:29:30 2231.84 km
23:55:56 2271.88 km
24:02:42 2283.27 km
24:27:16 2322.51 km
24:29:29 2325.95 km
24:34:00 2332.95 km
24:36:46 2337.23 km
24:39:32 2341.52 km
24:40:16 2342.66 km
24:48:33 2355.50 km
24:50:10 2358.00 km
24:54:39 2364.95 km
24:56:35 2367.95 km
24:59:12 2372.01 km
25:19:34 2396.12 km
25:20:02 2396.64 km
25:31:44 2418.08 km
25:33:02 2420.83 km
25:49:11 2446.37 km
25:53:44 2450.92 km
25:55:17 2451.83 km
26:00:19 2452.58 km
26:09:49 2454.39 km
26:16:13 2463.45 km
26:31:13 2485.56 km
26:37:22 2495.50 km
26:38:12 2496.85 km
26:41:35 2506.13 km
26:42:27 2508.85 km
26:42:59 2510.52 km
26:56:42 2552.42 km
27:12:53 2576.40 km
27:25:08 2594.93 km
27:47:14 2637.68 km
27:50:36 2642.84 km
27:52:11 2645.27 km
28:02:23 2657.72 km
28:18:49 2683.60 km
28:19:21 2684.55 km
28:28:21 2705.20 km
28:42:05 2719.04 km
28:54:44 2744.76 km
28:56:42 2751.28 km
28:57:20 2753.38 km
